python/ray/includes/common.pxd

```diff
@@ -120,3 +127,18 @@ cdef extern from "ray/common/task/scheduling_resources.h" \
         c_bool IsEmpty() const
         const unordered_map[c_string, double] &GetResourceMap() const
         const c_string ToString() const
+
+cdef extern from "ray/common/buffer.h" namespace "ray" nogil:
+    cdef cppclass CBuffer "ray::Buffer":
+        uint8_t *Data() const
+        size_t Size() const
+
+    cdef cppclass LocalMemoryBuffer(CBuffer):
+        LocalMemoryBuffer(uint8_t *data, size_t size) except +
```

Comment:
The `except +` declaration tells Cython to try to convert any C++ exceptions that are raised to corresponding Python ones. Removing this because the constructor shouldn't fail except in very exceptional circumstances and having it only here is more confusing than it's worth.